### ISSUE TITLE
User model issue

### ISSUE THREAD

**Issue by kikoanis** [lancetw/react-isomorphic-bundle#5]
User model associates posts model and userInfo model based on id foreign key. That will create a new constraint on both posts and userInfo tables that associate id to users table where it should be uid to users table. Hence this will cause a potential bigger issue of failure if you try to insert a post that points to non-existing user id or it may point to the wrong one.

These are the offensive lines
https://github.com/lancetw/react-isomorphic-bundle/blob/master/src/server/db/models/user.js#L32
https://github.com/lancetw/react-isomorphic-bundle/blob/master/src/server/db/models/user.js#L35

![capture](https://cloud.githubusercontent.com/assets/615849/10197348/568ff3ce-6754-11e5-8cec-1839b8ac048a.PNG)

they should pont to uid like so...
```
associate: function (models) {
        User.hasMany(models.posts, {
          foreignKey: 'uid'
        });
        User.hasOne(models.usersInfo, {
          foreignKey: 'uid'
        });
      }
```

**Comment by lancetw** [comment]
Thanks a lot for correct my mistake, also fix `userInfo` relations in dc520e7

**Comment by kikoanis** [comment]
Great thanks. Your fix might explain other issue I noticed but could not figure why they happen.
